echoregions/regions2d/regions2d.py

```diff
@@ -564,30 +571,89 @@ def transect_mask(
         ----------
         da_Sv : Data Array
             DataArray of shape (ping_time, depth) containing Sv data.
-        transect_dict : dict
-            Dictionary for transect values. Values must be unique.
+        transect_sequence_type_dict : dict
+            Dictionary for transect sequence type values. The values denote where in the context
+            of the transect each region lays in, i.e. whether we are at the beginning of the
+            transect, a break in the transect, a resumption of the transect, or at the end
+            of the transect.
+        transect_sequence_type_next_allowable_dict : dict
+            Dictionary for the allowable transect sequence type value(s) that can follow a
+            transect sequence type value.
         bbox_distance_threshold: float
-            The maximum value for how far apart the left and right bounding box for each transect
-            value region. Default is set to 1 minute.
+            Maximum allowable value between the left and right bounding box timestamps
+            for each transect value region. Default is set to 1 minute.
```

Comment:
Not sure if I get this right. It is for each region that is a transect log?
```suggestion
            Maximum allowable value between the left and right bounding box timestamps
            for each region that is marked as a transect log. Default is set to 1 minute.
```